### PR TITLE
34). Add topic files 103 to 106

### PR DESCRIPTION
**Pull request by jameshkramer**
This PR is for user story #1576037.

It adds four article files, makes corresponding deletions from the C# and VB folders, and adds redirection entries. It also makes some changes to toc.yml.

Please merge #18197, #18259, #18299, #18325, and #18413 before merging this.

cc: @BillWagner @mairaw @tfosmark @marteeleigh

Contributes to #4728

### CODE REVIEW

**Comment by mairaw** [comment]
Time for another merge conflict resolution @jameshkramer  😉

File: docs/standard/linq/toc.yml [hunk 1]
```diff
@@ -240,8 +240,8 @@
               href: chain-standard-query-operators-together.md
           - name: "Tutorial: Manipulate content in a WordprocessingML document"
             items:
-            - name: Shape of WordprocessingML documents
-              href: shape-wordprocessingml-documents.md
+            - name: The XML shape of WordprocessingML documents
+              href: xml-shape-wordprocessingml-documents.md
```
Comment (mairaw) [inline docs/standard/linq/toc.yml:244]: I see a couple of broken links in this PR for shape-wordprocessingml-documents.md. Can you make sure that existing links to the old file name are fixed?

File: docs/standard/linq/xml-shape-wordprocessingml-documents.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+# The XML shape of WordprocessingML documents (LINQ to XML)
+
+This article introduces the XML shape of a WordprocessingML document.
+
+## Microsoft Office formats
+
+The native file format for the 2007 Microsoft Office system is Office Open XML (commonly called Open XML). Open XML is an XML-based format that is an Ecma standard and is currently going through the ISO-IEC standards process. The markup language for word processing files within Open XML is called WordprocessingML. This tutorial uses WordprocessingML source files as input for the examples.
+
+If you are using Microsoft Office 2003, you can save documents in the Office Open XML format if you have installed the Microsoft Office Compatibility Pack for Word, Excel, and PowerPoint 2007 File Formats.
```
Comment (mairaw) [inline docs/standard/linq/xml-shape-wordprocessingml-documents.md:19]:
```suggestion
If you're using Microsoft Office 2003, you can save documents in the Office Open XML format if you have installed the Microsoft Office Compatibility Pack for Word, Excel, and PowerPoint 2007 File Formats.
```

File: docs/standard/linq/xml-shape-wordprocessingml-documents.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                Console.WriteLine(xDoc.Root)
+            End If
+        End Using
+    End Sub
+End Module
+```
+
+## See also
+
+- [Introducing the Office (2007) Open XML File Formats](https://docs.microsoft.com/previous-versions/office/developer/office-2007/aa338205%28v=office.12%29)
```
Comment (mairaw) [inline docs/standard/linq/xml-shape-wordprocessingml-documents.md:124]:
```suggestion
- [Introducing the Office (2007) Open XML File Formats](https://docs.microsoft.com/previous-versions/office/developer/office-2007/aa338205(v=office.12))
```

File: docs/standard/linq/xml-shape-wordprocessingml-documents.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+- [Overview of WordprocessingML](https://docs.microsoft.com/previous-versions/office/developer/office-2003/aa212812%28v=office.11%29)
```
Comment (mairaw) [inline docs/standard/linq/xml-shape-wordprocessingml-documents.md:125]:
```suggestion
- [Overview of WordprocessingML](https://docs.microsoft.com/previous-versions/office/developer/office-2003/aa212812(v=office.11))
```

File: docs/standard/linq/toc.yml (hunk 1 above)
Comment (jameshkramer) [reply docs/standard/linq/toc.yml:244]: Done - pushed another commit

**Pull request merged**